demo-app/README.md

```diff
@@ -10,12 +10,14 @@ This will ensure you have everything you need installed and boot up the developm
 script/server
 ```
 
-The application will be available at [http://localhost:3000](http://localhost:3000).
+The application will be available at [http://localhost:3000](http://localhost:3000) and the GraphiQL UI will be available at [http://localhost:3000/api/graphql](http://localhost:3000/api/graphql).
 
 ## Development
 
 Model Home is a mix of Open Truss examples and research we're doing that has not yet been added to an `@open-truss` package.
 
+When making changes to `src/types/schema.graphql` make sure to run `npm run codegen` to regenerate Typescript types from the GraphQL SDL.
```

Comment:
Does it make sense to add this to script/server?

Reply:
Alternatively if it doesn't make sense there, does the system give you any warnings/errors if it detects outdated values from the GraphQL SDL? Thinking things like "It looks like you need to regenerate xyz".

Reply:
Great feedback. Documented in https://github.com/open-truss/open-truss/issues/173